### DISCUSSION TITLE
요번 리뷰는 내일 올릴 예정입니다..

### DISCUSSION THREAD

**Discussion by Joseph-Cha** [ThinkAboutSoftware/AcademicConference#192] [General]
당장 오늘 있었던 내부 시연 때문에 요 며칠 야근을 밥 먹듯이 하고 있었습니다...(라는 핑계를 일단 밑밥으로 깝니다ㅇㅁㅇ)
다행히 시연이 잘 마무리 되어서 내일 휴가를 얻었는데요, 내일 부지런히 읽어서 리뷰를 올리겠습니다..!

**Comment by jongfeel** [comment]
야근을 하셨군요. 그래도 내일 올려주신다고 하니 다행입니다.
너무 무리하지는 마시고 할 수 있는 부분 까지라도 하시면 좋을 것 같습니다.
